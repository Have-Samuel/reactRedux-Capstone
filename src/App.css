* {
  padding: 0;
  margin: 0;
  box-sizing: border-box;
}

img {
  max-width: 100%;
}

body {
  background-color: rgb(179, 15, 143);
  color: #fff;
}

.accordion {
  background-color: rgb(126, 17, 80) !important;
  padding: 10px 2px;
}

.accordion-eader > .accordion-button {
  background-color: rgb(133, 9, 112);
  color: #fff;
}

h1,
h2,
h3,
h4,
h5,
h6 {
  padding: 0;
  margin: 0;
}

h3 {
  font-size: 18px;
}

ul {
  padding: 0;
}

li {
  display: flex;
  width: 100%;
  justify-content: space-between;
  padding: 0 10px;
}

ul li {
  list-style: none;
  padding: 0;
  margin: 0 auto;
  width: 100%;
}

small {
  overflow: hidden;
  text-overflow: ellipsis;
  margin-left: 10px;
  color: antiquewhite;
}

.location,
.diet {
  font-size: 12px;
}

.factName {
  margin-right: 30px;
  font-size: 20px;
}

.factProp {
  font-size: 17px;
}

.facts > li {
  padding: 15px 10px;
}

.facts > li:nth-child(odd) {
  background-color: rgba(190, 18, 182, 0.6);
}

@media screen and (min-width: 768px) {
  small {
    text-overflow: unset;
  }
}
