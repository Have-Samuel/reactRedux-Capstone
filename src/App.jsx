/* eslint-disable react/jsx-no-undef */
import React from 'react';
import { BrowserRouter as Router, Route, Routes } from 'react-router-dom';
import Details from './components/pages/Details';
import Home from './components/pages/Home';

function App() {
  return (
      <Router>
        <Routes>
          <Route path="/" element={<Home />} />
          <Route path="/details" element={<Details />} />
        </Routes>
      </Router>
  );
}

export default App;
