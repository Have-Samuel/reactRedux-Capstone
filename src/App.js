import React from 'react';
import { BrowserRouter as Router, Route, Routes } from 'react-router-dom';
import './App.scss';
import Navbar from './components/common/Navbar';
import Rockets from './routes/Rockets';

function App() {
  return (
    <Router>
      <Navbar />
      <Routes>
        <Route path="/" element={<Rockets />} />
      </Routes>
    </Router>
  );
}

export default App;
