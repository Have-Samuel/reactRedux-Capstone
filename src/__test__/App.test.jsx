import React from 'react';
import renderer from 'react-test-renderer';
import { Provider } from 'react-redux';
import App from '../App';
import store from '../redux/configureStore';

describe('render App correctly', () => {
  test('render', () => {
    const value = renderer.create(
        <Provider store={store}>
          <App />
        </Provider>
    );
    expect(value).toMatchSnapshot();
  });
});
