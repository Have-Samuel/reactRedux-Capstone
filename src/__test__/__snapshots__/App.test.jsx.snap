// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render App correctly render 1`] = `
Array [
  <header
    className="header"
  >
    <nav
      className="nav"
    >
      <img
        alt="animal"
        className="nav-icons"
        src="./images/Pearl.png"
      />
      <h2
        className="nav-title"
      >
        Respect For Wildlife
      </h2>
      <div
        className="nav-search"
      >
        <img
          alt="mic"
          className="nav-icons mic"
          src="./images/mic.png"
        />
        <img
          alt="mic"
          className="nav-icons"
          src="./images/settings-cog.png"
        />
      </div>
    </nav>
    <img
      alt="Img"
      className="home-img"
      src="images/Pearl.png"
    />
    <div
      className="image-cover"
    />
  </header>,
  <div
    className="search-sec"
  >
    <h4>
      Animal Species
    </h4>
    <input
      className="input"
      onChange={[Function]}
      palceholder="Search Animal..."
      type="text"
    />
  </div>,
  <main
    className="main"
  />,
]
`;
